### DISCUSSION TITLE
Add a version displayed at the bottom of the page

### DISCUSSION THREAD

**Discussion by Seluj78** [shellhub-io/shellhub#3505] [Ideas]
As a user that actively reports bugs, I would like to have the version displayed in a footer of the UI or somewhere easily findable so I can find it quickly (even though I can just ssh into the shellhub instance and check the git tag)

**Comment by heiytor** [comment]
Unfortunately, the UI does not display the ShellHub version. However, you can make a request to [/info](https://docs.shellhub.io/developers/openapi/community#operation/getInfo) to retrieve information about your instance.

**Reply by Seluj78** [reply]
Yeah I figured, but I just suggested this so that users can more easily submit bugs by finding the version quicker
